crates/extensions/tedge_mqtt_ext/src/lib.rs

```diff
@@ -187,36 +234,30 @@ impl Actor for MqttActor {
     }
 
     async fn run(&mut self) -> Result<(), RuntimeError> {
+        let (mut inbox, outbox) = self
```

Comment:
```suggestion
        let (mut peers_to_mqtt, mqtt_to_peers) = self
```
I find the names `inbox` and `outbox` a bit confusing. From this actor's perspective, it receives inputs from 2 different sources: The MQTT broker and peer actors. Similarly, it pushes its output also to both of them. So when you call something an `inbox`, I'm always asking in my head, is it the `inbox` for the MQTT messages from the broker or the the `inbox` for MQTT messages from peers (NOTE: it's the same `MqttMessage` type from/to both sources). Similarly I question `outbox` as well. The problem is that the words "in" and "out" are vague here: "In" from peers or "in" from broker.

Here, the `inbox` reads from the actors and forwards to MQTT and `outbox` reads from MQTT and forwards that to peer actors.

Reply:
I've renamed this, but tried to limit the number of times we mention peers and mqtt (with all of the suggestions, the main usage will involve mentioning both at least twice. Also `peers_to_mqtt` isn't quite true; in one place we use it just to receive the shutdown signal, which isn't "to mqtt". As a result, I've called these `from_peers` and `to _peers`